web/src/beta/lib/core/StoryPanel/ActionPanel/index.tsx

```diff
@@ -38,12 +39,22 @@ type Props = {
     schemaItemId?: string,
     fieldId?: string,
     itemId?: string,
-    vt?: string,
-    v?: any,
+    vt?: ValueType,
+    v?: ValueTypes[ValueType],
```

Comment:
In core these should be set to any for now. Don't want any reference to graphql for now to avoid breakage on published projects 🙏🏻

Reply:
done